backend/tests/v5/unit/services/mailer/mailer.test.js

```diff
@@ -61,6 +61,7 @@ const testSendEmail = () => {
 		test('should fail if config.mail.smtpConfig is not set and config.mail.generateCredentials is false', async () => {
 			Mailer.reset();
 			config.mail.generateCredentials = false;
+			delete config.mail.smtpConfig;
 			await expect(Mailer.sendEmail(emailTemplates.FORGOT_PASSWORD.name, recipient, data, attachments))
 				.rejects.toThrow('config.mail.smtpConfig is not set');
 			config.mail.generateCredentials = true;
```

Comment:
I would keep the config in the same state after each test. Could we not set generateCredentials to true by default ?

Reply:
This line is actually redundant because we reset the config before each test: https://github.com/3drepo/3drepo.io/blob/9b91d06dd362fc1f72de785a7f65b0356750991c/backend/tests/v5/unit/services/mailer/mailer.test.js#L47

I'll remove it

Reply:
ah true